docs/wiki/Fetching latest changes - Scenario 1 Module library only.md

```diff
@@ -0,0 +1,99 @@
+When the CARML library is consumed with the [*Template Orchestration*](./Solution%20creation#template-orchestration) approach modules need to be stored in an available location, where the Azure Resource Manager (ARM) can access them. This can be achieved by storing the modules templates in an accessible location like local, Template Specs or the Bicep Registry.
+
+---
+### _Navigation_
+
+- [Local Repository](#local-repository)
+- [Template Specs](#template-specs)
+- [Bicep Registry](#bicep-registry)
+
+---
+
+# Local Repository
+In case you consume the module library with a local repository, the update procedure has the following steps:
+1. Backup your local repository to maintain a backup copy
```

Comment:
It looks like all the steps besides the last one (publishing) are the same for all options, i.e. backup, download, conversion and customization. I'd suggest to describe them just once for maintainability and then differentiate the last step (publishing) based on the approach.

Reply:
grouped by procedure instead of repository